### PR TITLE
Add support for 'nullable' properties

### PR DESCRIPTION
**Pull request by jayniz**
For examples properties such as

``` json
  {
    "type": [
      {"type": "null"},
      {
        "type": "object",
        "properties": { ... }
      }
    ]
  }
```

### CODE REVIEW

File: lib/blumquist.rb [hunk 1]
```diff
@@ -112,7 +178,6 @@ def blumquistify_array(property)
     # The items of this array are defined by a pointer
     if type_def[:$ref]
       item_schema = resolve_json_pointer!(type_def)
-      raise(Errors::MissingArrayItemsType, @schema[:properties][property]) unless item_schema
```
Comment (thinkberg) [inline lib/blumquist.rb]: why is this removed?

Comment (jayniz) [inline lib/blumquist.rb]: Can't be reached because `resolve_json_pointer!` will raise in that case
